--- gnome-abrt-1.2.0/src/gnome_abrt/wrappers/module.c~	2015-06-03 05:36:53.000000000 +0200
+++ gnome-abrt-1.2.0/src/gnome_abrt/wrappers/module.c	2015-09-18 09:37:15.949588371 +0200
@@ -41,7 +41,11 @@
             .m_doc = module_doc,
             .m_size = -1,
             .m_methods = module_methods,
+#if PY_MAJOR_VERSION >= 3 && PY_MINOR_VERSION >= 5
+            .m_slots = NULL,
+#else
             .m_reload = NULL,
+#endif
             .m_traverse = NULL,
             .m_clear = NULL,
             .m_free = NULL,
