--- gnome-abrt-0.2.10/src/gnome-abrt.orig	2013-03-26 13:28:58.000000000 +0100
+++ gnome-abrt-0.2.10/src/gnome-abrt	2013-04-07 13:40:20.995848644 +0200
@@ -71,7 +71,7 @@
             self._app = application
 
         @dbus.service.method(dbus_interface=GNOME_ABRT_INTERFACE,
-                             in_signature='as', out_signature='')
+                             in_signature = 'as', out_signature = '')
         def command_line(self, argv):
             """DBus service method accepting a new command line arguments
             """
